data/persistence/src/androidMain/kotlin/social/androiddev/common/persistence/di/AndroidPersistenceModule.kt

```diff
@@ -44,4 +45,13 @@ actual val persistenceModule: Module = module {
         )
         AuthenticationDatabase(driver)
     }
+
+    single {
+        val driver = AndroidSqliteDriver(
+            schema = TimelineDatabase.Schema,
+            context = get(),
+            name = FEED_DB_NAME,
+        )
+        TimelineDatabase(driver)
```

Comment:
Do we want to cache the timeline? This is to support offline usage? Maybe it will become more clear later in the PR diff, but on the surface it seems this table/db will be REALLY large, especially for federated timeline. Also, is federated and local timeline stored in separate DB/tables?

Reply:
Yup the goal is to be an offline first app and cache aggressively. We can expire/cleanup the DB table if it gets large but TBH feed data should be pretty small. 

As for storing together or separate. My current thinking is to have a column for FeedType when pulling posts/statuses and store in the same table but I haven't given data modeling much thought, open to any and all suggestions <3

Reply:
I see ok, so we would just be caching some subset of the entire timeline, and having that expire/update as user scrolls when online? Might be worth considering separating the table for federated vs home timelines. Then you don't need to do a query based on some filter each time, as well as potentially get into a case where some fields are only present on federated vs home timeline and having to enforce type-safety when mapping DB -> domain models based on a FeedType etc

Reply:
yup makes sense, will make separate tables when building the next feed